src/System.Diagnostics.Process/src/Resources/Strings.resx

```diff
@@ -312,4 +312,79 @@
   <data name="StandardInputEncodingNotAllowed" xml:space="preserve">
     <value>StandardInputEncoding is only supported when standard input is redirected.</value>
   </data>
+  <data name="IO_AlreadyExists_Name" xml:space="preserve">
+    <value>Cannot create '{0}' because a file or directory with the same name already exists.</value>
+  </data>
+  <data name="IO_BindHandleFailed" xml:space="preserve">
```

Comment:
Is `GetExceptionForIoErrno` ever going to produce a good diagnosable error for `InitializeSignalHandling`? It seems like that these errors are specifically designed for file I/O.